tests/testdata/aws-vpc/go/go.mod

```diff
@@ -0,0 +1,10 @@
+module aws-vpc
+
+go 1.20
+
+require (
+	github.com/pulumi/pulumi/sdk/v3 v3.30.0
+	github.com/pulumi/pulumi-terraform-module/sdk/go/v5 v5.18.1
+)
+
+replace github.com/pulumi/pulumi-terraform-module/sdks => ./sdks/vpc
```

Comment:
What happens if you have two modules in one program, vpc AND bucket? Do they clobber and conflict on this or they work out ok?

```
github.com/pulumi/pulumi-terraform-module/sdks
```

Reply:
they should not because in the program they are required as `github.com/pulumi/pulumi-terraform-module/sdks/bucket` and `github.com/pulumi/pulumi-terraform-module/sdks/vpc` respectively, but it's worth verifying

Reply:
I take this back - the generated go.mod inside  is just named github.com/pulumi/pulumi-terraform-module/sdks; that should be parameterized by module name. will fix.